docs/source/llms.txt.template

```diff
@@ -0,0 +1,148 @@
+# SkyPilot
+
+SkyPilot is a system for running, managing, and scaling AI workloads on any AI infrastructure.
+
+SkyPilot gives AI teams a simple interface to run jobs on any infra. Infra teams get a unified control plane to manage any AI compute — with advanced scheduling, scaling, and orchestration.
+
+## Getting Started
```

Comment:
For the titles, should we follow the TOC's titles/structure?

Reply:
Also, we should emphasize the API server / team deployment.

Reply:
> should we follow the TOC's titles/structure?

I updated it to closer follow the TOC. 

>Also, we should emphasize the API server / team deployment.

done!